apps/modernization-ui/src/apps/search/investigation/result/list/table/InvestigationSearchResultsTable.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import { render } from "@testing-library/react";
+import { InvestigationSearchResultsTable } from "./InvestigationSearchResultsTable";
+import { Investigation } from "generated/graphql/schema";
+import { SearchResultDisplayProvider } from 'apps/search/useSearchResultDisplay';
+import { MemoryRouter } from 'react-router-dom';
+import { SkipLinkProvider } from 'SkipLink/SkipLinkContext';
+import { ColumnPreferenceProvider } from "design-system/table/preferences";
+
+describe('When InvestigationSearchResultsTable renders', () => {
+    const testResults: Investigation[] = [{
+        "__typename": "Investigation",
+        "relevance": 1,
+        "id": "10000013",
+        "cdDescTxt": "Pertussis",
+        "jurisdictionCodeDescTxt": "Clayton County",
+        "localId": "CAS10000000GA01",
+        "addTime": "2015-09-22",
+        "investigationStatusCd": "O",
+        "notificationRecordStatusCd": "UNASSIGNED",
+        "personParticipations": [
+            {
+                "__typename": "InvestigationPersonParticipation",
+                "birthTime": "1990-01-01",
+                "currSexCd": "M",
+                "typeCd": "SubjOfPHC",
+                "firstName": "Surma",
+                "lastName": "Singh",
+                "personCd": "PAT",
+                "personParentUid": 10000001,
+                "shortId": 63000
+            }
+        ]
+    }];
+
+    const Wrapper = () => {
+        return (
+            <MemoryRouter>
+                <SkipLinkProvider>
+                    <SearchResultDisplayProvider>
+                        <ColumnPreferenceProvider>
+                            <InvestigationSearchResultsTable results={testResults} />
+                        </ColumnPreferenceProvider>
+                    </SearchResultDisplayProvider>
+                </SkipLinkProvider>
+            </MemoryRouter>
+        )
+    }
+    
+    it('should display 9 columns', () => {
+        const { container } = render(<Wrapper />)
+        const columns = container.getElementsByTagName('th');
+        expect(columns).toHaveLength(12);
+    });
```

Comment:
Issue: Test says it should display 9 columns but then verifies there are 12 columns.